src/lambdas/ingest/index.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/prefer-default-export */
 import got from 'got' // eslint-disable-line import/no-unresolved
 import { createIndex } from '../../lib/database-client.js'
-import { ingestItems, publishResultsToSns } from '../../lib/ingest.js'
+import { processMessages, publishResultsToSns } from '../../lib/ingest.js'
```

Comment:
renamed to better indicate that there were multiple types of messages it was expecting. This was already an inaccurate name, as Collections could be ingested in addition to Items already.